samples/subsys/usb/hid/src/main.c

```diff
@@ -81,19 +83,45 @@ int get_report_cb(struct usb_setup_packet *setup, s32_t *len,
 	return 0;
 }
 
+static void in_ready_cb(void)
+{
+	SYS_LOG_DBG("");
+
+	usb_transfer_ep_callback(0x81, USB_DC_EP_DATA_IN);
+}
+
+static void hid_write_cb(u8_t ep, int size, void *priv)
+{
+	SYS_LOG_DBG("ep %x size %d", ep, size);
+
+	usb_transfer(ep, report_1, sizeof(report_1),
+		     USB_TRANS_WRITE | USB_TRANS_NO_ZLP,
+		     hid_write_cb, NULL);
+}
+
+static void status_cb(enum usb_dc_status_code status, u8_t *param)
+{
+	switch (status) {
+	case USB_DC_CONFIGURED:
+		hid_write_cb(0x81, 0, NULL);
+	default:
```

Comment:
In that case you can just call `in_ready_cb` instead to start the transfer loop.